src/services/fetchers/staking-transaction-fee-reserve-fetcher.ts

```diff
@@ -0,0 +1,16 @@
+import { newMonetaryAmount } from "@interlay/interbtc-api";
+import { AddressOrPair } from "@polkadot/api/types"
+
+import { GovernanceTokenMonetaryAmount, GOVERNANCE_TOKEN } from "config/relay-chains";
+
+const getStakingTransactionFeeReserve = (account: AddressOrPair) => async (): Promise<GovernanceTokenMonetaryAmount> => {
```

Comment:
Naming could be adjusted as other fetchers for consistency. Not a strong opinion though.